plugins/out_stackdriver/stackdriver_conf.c

```diff
@@ -286,6 +296,33 @@ struct flb_stackdriver *flb_stackdriver_conf_create(struct flb_output_instance *
         ctx->severity_key = flb_sds_create(tmp);
     }
 
+    if (flb_sds_cmp(ctx->resource, "k8s_container", 
```

Comment:
[Optional] 

This means you can only use the output plugin for one type of resources. Is it your intention? If yes, I think I don't have concern.

I think this is fine for me as in some context you may only want to use fluent-bit to send specific resource type. I expect you can use single fluent-bit for multiple k8s resources though but it can be outside the range of this PR. in that case, you can  get the `cluster_location` and `cluster_name` from config without checking the resource and the configurations can be used for all k8s workloads.

Reply:
Hi Yen-Cheng I think the scope of this patch will support specific resource type first. This could be scaled up and support multiple resource types using one fluent-bit in the future. Thanks.

Reply:
SGTM.

Reply:
Good. Thanks